assets/src/edit-story/karma/fixture/containers/checklist.js

```diff
@@ -52,10 +52,18 @@ export class Checklist extends Container {
     return this.queryByRole('tabpanel', { name: /^Priority/ });
   }
 
+  get expandedPriorityTab() {
+    return this.getByRole('tab', { name: /Priority issue/, selected: true });
+  }
+
   get designTab() {
     return this.getByRole('tab', { name: /Design issue/ });
   }
 
+  get expandedDesignTab() {
+    return this.getByRole('tab', { name: /Design issue/, selected: true });
+  }
+
```

Comment:
I needed a way to check to see which tab was actually selected.

I don't love that I added these to the fixture but I figured this was equally as ugly as writing this in the test directly. It's repeated code though, and I'm happy to remove it.

I figured that it made more sense here because the `name` was the same on each. Thoughts?

Reply:
i like this here because otherwise we repeat it - i was repeating this alot fixing the existing karma tests that i pulled in from the previous structure in #8174